interfaces/builtin/mpris.go

```diff
@@ -88,6 +88,31 @@ dbus (receive)
     bus=session
     path=/org/mpris/MediaPlayer2
     peer=(label=@{profile_name}),
+
+# Allow unconfined session to interact with the player
+dbus (receive)
+    bus=session
+    interface=org.freedesktop.DBus.Properties
+    path=/org/mpris/MediaPlayer2
+    peer=(label=unconfined),
+
+dbus (receive)
+    bus=session
+    interface=org.freedesktop.DBus.Introspectable
+    peer=(label=unconfined),
+
+dbus (receive)
```

Comment:
there's a similar bit in mprisConnectedSlotAppArmorClassic although it doesn't specify the interface for some reason.

@alexmurray looks like we may not need the mprisConnectedSlotAppArmorClassic snippet, do we?